POC/ColorPicker/ColorPicker/ColorPicker.csproj

```diff
@@ -94,5 +98,6 @@
   <ItemGroup>
     <None Include="App.config" />
   </ItemGroup>
+  <ItemGroup />
```

Comment:
This line seems like it doesn't do anything, should be safe to remove

Reply:
Good point, will remove 👍